packages/common/src/doctype.ts

```diff
@@ -18,56 +18,85 @@ export enum AnchorStatus {
     NOT_REQUESTED, PENDING, PROCESSING, ANCHORED, FAILED
 }
 
-/**
- * Describes anchor record
- */
-export interface AnchorRecord {
-    prev: CID; // should be CID type
-    proof: CID; // should be CID type
-    path: string;
+export interface RecordHeader {
+    controllers: Array<string>
+    schema?: string
+    tags?: Array<string>
+
+    unique?: string
+
+    [index: string]: any // allow support for future changes
+}
+
+export type GenesisHeader = RecordHeader
+
+export type GenesisRecord = {
+    header: GenesisHeader
+    data?: any
+    unique?: string
+}
+
+export interface UnsignedRecord {
+    id: CID
```

Comment:
what CID is this?  Is this the records own CID?  That couldn't ever be in the record itself...

EDIT: I believe this is actually the CID of the genesis record.  Basically a proxy for the DocID.  Can we just add a quick comment clarifying that?